src/Exception/OgRoleRequiredException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Drupal\og\Exception;
+
+/**
+ * Exception thrown when a required role is compromised.
```

Comment:
mayeb `Exception thrown on invalid deletion or invalid set of immutable properties.`